plugins/metadata/lib/kMetadataKavaUtils.php

```diff
@@ -89,8 +93,10 @@ protected static function getMetadataFieldValues($objectIds, $partnerId, $metada
 				}
 				else
 				{
-					$fullPath = realpath($fileSync['FILE_ROOT'] . $fileSync['FILE_PATH']);
+					$fullPath = kFile::realPath($fileSync['FILE_ROOT'] . $fileSync['FILE_PATH']);
```

Comment:
Better use kFile::getFileContent
Which supports both cases (file on NFS stoarge or file stored on S3).

Reply:
Thanks for the tip!
Done (and tested)